test/functional/proposer_balance.py

```diff
@@ -60,7 +60,9 @@ def test_empty_blocks_balance(self, created_money, nodes):
             coinstake_tx_id = block_info['tx'][0]
             coinstake_tx_info = nodes[node_idx].gettransaction(coinstake_tx_id)
 
-            created_money += coinstake_tx_info['details'][0]['amount']
+            # TODO UNIT-E: gettransaction should not return fees for coinbase transactions
```

Comment:
I would like to have an issue here as well. We don't want to forget this stuff.

Reply:
Did we create an issue for this?

Reply:
Not sure, can't find anything.